rewrite-weblogic/src/main/resources/META-INF/rewrite/jakarta-ee-9.1.yaml

```diff
@@ -337,4 +338,223 @@ recipeList:
   - org.openrewrite.java.dependencies.UpgradeDependencyVersion:
       groupId: org.glassfish.jersey.media
       artifactId: '*'
-      newVersion: 3.x
\ No newline at end of file
+      newVersion: 3.x
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: com.oracle.weblogic.rewrite.jakarta.UpgradePluginConfigurationArtifacts
+displayName: Change artifacts for a Maven plugin configuration
+description: Change artifacts for a Maven plugin configuration artifacts.
+tags:
+  - jakarta
+  - dependencies
+recipeList:
+  - com.oracle.weblogic.rewrite.jakarta.UpgradePluginConfigurationArtifactItems:
+      oldGroupId: javax
+      oldArtifactId: javaee-api
+      newGroupId: jakarta.platform
+      newArtifactId: jakarta.jakartaee-api
+      newVersion: 9.1.0
+  - com.oracle.weblogic.rewrite.jakarta.UpgradePluginConfigurationArtifactItems:
+      oldGroupId: javax
+      oldArtifactId: javaee-web-api
+      newGroupId: jakarta.platform
+      newArtifactId: jakarta.jakartaee-web-api
+      newVersion: 9.1.0
+  - com.oracle.weblogic.rewrite.jakarta.UpgradePluginConfigurationArtifactItems:
```

Comment:
how can we be sure that the "newVersion" values in this recipe work with whatever Maven plugin the user is using?

Reply:
The new versions defined for the recipe are the versions of Jakarta EE components in 9.1.